### PR TITLE
199 responsive images

### PR DESCRIPTION
**Pull request by michaelKurowski**

### CODE REVIEW

**Comment by michaelKurowski** [comment]
`http://localhost:8000/articles/image/deno-things-adsdadas?width=100` passing width parameter returns full-sized image instead of scaled one @Jordan141

**Comment by michaelKurowski** [comment]
```
if(width && height) return getArticleImage(res, req.params.link, width, height).catch(err => req.log(err))
    return getArticleImage(res, req.params.link).catch(err => req.log(err))
    ```
    Apperently endpoints require both width and height. I need smart scalling, as I can't predict proportions of an image.

**Comment by michaelKurowski** [comment]
Works in Firefox, but Chromes does some black magic and it doesn't work there yet.

**Comment by michaelKurowski** [comment]
526 mapps to 1024 on Chrome, while 651 to 2048 for some reason

**Comment by michaelKurowski** [comment]
1024 maps to 2049

**Comment by michaelKurowski** [comment]
2048 to 2049, looks like Chrome always catches one size higher than asked for.

**Comment by michaelKurowski** [comment]
It might have sth to do with `devicePixelRatio` returning `2`

**Comment by michaelKurowski** [comment]
DPR in dev tools was set to 2, which was the cause for that.

File: app/utils.js [hunk 1]
```diff
@@ -66,9 +66,13 @@ async function __getImage(res, imageName, folder, width, height) {
         if(!fs.existsSync(filePath)) throw new Error(`__getImage Error: ${filePath} does not exist`)
         const imageBuffer = await fs.promises.readFile(filePath)
         res.set('Content-Type', 'image/jpeg')
-        
-        if(width && height) return await sharp(imageBuffer).resize(parseInt(width), parseInt(height)).toFormat(JPEG).jpeg(JPEG_OPTIONS).pipe(res)
-        return await sharp(imageBuffer).toFormat(JPEG).jpeg(JPEG_OPTIONS).pipe(res)
+        let image = null
+        console.log(width, height)
```
Comment (Jordan141) [inline app/utils.js]: console log

**Review by Jordan141 — APPROVED** [review]

**Pull request merged**